lib/logstash/inputs/couchdb_changes.rb

```diff
@@ -0,0 +1,217 @@
+  # The heartbeat is how often (in milliseconds) Logstash will ping
+  # CouchDB to ensure the connection is maintained.  Changing this 
+  # setting is not recommended unless you know what you are doing.
+  config :heartbeat, :validate => :number, :default => 1000
+
+  # Where to write the sincedb database (keeps track of the last
+  # sequence of in the _changes stream). The default will write
+  # to a file matching "$HOME/.couchdb_seq"
+  config :sincedb_path, :validate => :string
+
+  # If unspecified, Logstash will attempt to read the last sequence number
+  # from the sincedb file.  If that is empty or non-existent, it will 
+  # begin with 0 (the beginning).
+  # 
+  # If you specify this value, it is anticipated that you will 
+  # only be doing so for an initial read under special circumstances
+  # and that you will change this value back to nil after that
+  # special ingest.
+  config :initial_sequence, :validate => :number, :default => nil
+  
+  # Preserve the CouchDB document revision "_rev" value in the
+  # output.
+  # Default: false
+  config :keep_revision, :validate => :boolean, :default => false
+  
+  # Future feature! Until implemented, changing this from the default 
+  # will not do anything.
+  #
+  # Ignore attachments associated with CouchDB documents. Default: True
+  config :ignore_attachments, :validate => :boolean, :default => true
+  
+  # Add type, tags, and add_field values to each event.
+  #
+  # Disabling this may only be desirable if you are passing all 
+  # documents straight through to Elasticsearch with no filters
+  # Default: true
+  config :decorate_event, :validate => :boolean, :default => true
+  
+  # Reconnect flag.  When true, always try to reconnect after a failure
+  # Default: true
+  config :always_reconnect, :validate => :boolean, :default => true
+  
+  # Reconnect delay: time between reconnect attempts, in seconds.
+  # Default: 10
+  config :reconnect_delay, :validate => :number, :default => 10
+  
+  # Timeout: Number of milliseconds to wait for new data before
+  # terminating the connection.  If a timeout is set it will disable
+  # the heartbeat configuration option.
+  # Default: nil
+  config :timeout, :validate => :number
+
+  public
+  def register
+    require "logstash/util/buftok"
+    if @sincedb_path.nil?
+      if ENV["HOME"].nil?
+        @logger.error("No HOME environment variable set, I don't know where " \
+                      "to keep track of the files I'm watching. Either set " \
+                      "HOME in your environment, or set sincedb_path in " \
+                      "in your Logstash config.")
+        raise 
+      end
+      sincedb_dir = ENV["HOME"]
+      @sincedb_path = File.join(sincedb_dir, ".couchdb_seq")
+
+      @logger.info("No sincedb_path set, generating one...",
+                   :sincedb_path => @sincedb_path)
+    end
+
+    @sincedb      = SinceDB::File.new(@sincedb_path)
+    @feed         = 'continuous'
```

Comment:
This was left mutable before a determination to only use `continuous` was made.  This can be changed to a constant.